docs/Governance.md

```diff
@@ -0,0 +1,223 @@
+lead maintainer with sysadmin work; this involves working to keep Homebrew's CI
+up-to-date, Homebrew's hardware secure and ensuring the Homebrew
+[website](https://brew.sh) remains up-to-date with TLS configuration &
+certification.
+
+The PLC is a formal body that primarily operates to communicate & resolve, with
+the Software Freedom Conservancy, any legal & administrative issues that may
+arise. Due to the legal involvement of this body members of the PLC cannot be
+anonymous, and must be willing to submit various details, including their real
+name, to the Software Freedom Conservancy.
+
+The PLC also has two more technical roles. The first of these roles is to vote
+on issues that cannot be resolved in regular discussion between maintainers,
+such as whether to approve a controversial idea or feature. PLC votes on
+technical issues should be used sparingly, reserved normally for issues that
+will regularly impact the vast majority of maintainers.
+
+The second of these roles is to provide checks & balances. The PLC as a body has
+an obligation to ensure power is being exercised carefully & fairly by all
+maintainers, including the lead maintainer. The lead maintainer must always be
+part of the PLC, but [cannot always](#accountability) vote on every issue that
+arises.
+
+### Regular Maintainers
+
+Regular maintainers are people who have commit access to Homebrew but are not
+expected or obligated to help with legal, technical or administrative issues
+that arise impacting the project itself. Regular maintainers can be members of
+multiple GitHub repositories or may only be maintainers of one repository, such
+as `homebrew/livecheck`.
+
+Regular maintainers spend almost all of their time around Homebrew providing
+code review or submitting pull requests of their own, although they should
+always prioritise the work of others above their own, with rare exceptions such
+as urgent security fixes.
+
+Regular maintainers do not routinely have access to Homebrew's hardware
+_(but can be granted access)_ and are not expected to do sysadmin work. Regular
+maintainers have access to CI for the purposes of build queue control &
+triggering jobs, and have the ability to manually request a rebuild via
+`@BrewTestBot test this please`.
+
+## Accountability
+
+Everyone is accountable for their words & actions, or lack thereof, in Homebrew,
+and nobody has unchecked power or influence. The below list covers various
+situations that may arise and how to handle them.
+
+### The Lead Maintainer Position & The Holder Of That Position
+
+At least once a year, provisionally September 14th, the PLC is mandated to
+affirm or withhold its consent for the lead maintainer role to exist in its
+current form as a distinct entity. The lead maintainer may vote on this issue.
+In the event of a tie regular maintainers are asked to vote on the issue and
+collectively act as the sole tie-breaking vote.
```

Comment:
I'm not sure voting once a year on this specific item is of particular interest. Maybe a more open question is: how does the governance system described here evolve over time? This includes the lead maintainer position, but it's not restricted to that, there are more aspects that could change over time, as is seen necessary.

Reply:
> there are more aspects that could change over time, as is seen necessary.

I note near the top this document can be modified by any simple-majority PLC vote, which allows for it to be a kind of living document rather than something that always has to be the way that it is now. It would be a case of someone simply suggesting something to the PLC, the PLC voting on it & making any necessary changes according to that vote.

I'm not sure if that answers your question or whether I'm misunderstanding slightly?